app/src/main/java/com/example/stardewvalleypockettracker/MainActivity.java

```diff
@@ -1,14 +1,27 @@
 package com.example.stardewvalleypockettracker;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
 
 import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuInflater;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Toolbar toolbar = findViewById(R.id.toolbar);
```

Comment:
We might be able to use a base component to do this.  I shared a potential solution in discord.